docker/entrypoint-validator.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+HOME_DIR="$HOME/.arda-poc"
+
+if [ ! -f "$HOME_DIR/config/genesis.json" ]; then
+  arda-pocd init "$MONIKER" --chain-id "$CHAIN_ID" --home "$HOME_DIR"
+  arda-pocd config keyring-backend test --home "$HOME_DIR"
+  arda-pocd keys add "$MONIKER" --keyring-backend test --home "$HOME_DIR" --algo secp256k1 --yes
+  arda-pocd add-genesis-account $(arda-pocd --home "$HOME_DIR" keys show "$MONIKER" -a --keyring-backend test) 1000000000uarda --home "$HOME_DIR"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)